### ISSUE TITLE
Conversion may fail sometimes due to too many concurrent sessions

### ISSUE THREAD

**Issue by valkjsaaa** [valkjsaaa/auto-bilibili-recorder#9]
```
[h264_nvenc @ 0x5601aa5b40c0] OpenEncodeSessionEx failed: out of memory (10)
[h264_nvenc @ 0x5601aa5b40c0] No NVENC capable devices found
```

**Comment by Alexadnar** [comment]
u used the GPU version of this docker, pls change ur docker images to the CPU only one
and ur memoriy of computer is too small

**Comment by valkjsaaa** [comment]
this is fixed with [d76777d](https://github.com/valkjsaaa/auto-bilibili-recorder/commit/d76777dfc94e08e7928bfe975f5bb3e82bd6b6a2)
